### ISSUE TITLE
11.5.2.10 Textinhalte und –Attribute zugänglich - Entwurf Prüfung

### ISSUE THREAD

**Issue by detlevhfischer** [BIK-BITV/BIK-App-Test#43]
Hallo,
in erster Linie wird hier geprüft, ob Text **überhaupt** ausgegeben wird, oder als Text-layer für Screenreader völlig unzugänglich ist.

Des Weiteren soll aber auch das Auslesen von Textauszeichnungen überprüft werden. ich habe in #42 eine Prüfung in iOS und Android skizziert, die aber recht komplex ist und wohl ohnehin nur praktikabel ist, wenn die Software / App diese Ooptionen bietet. In Pages (iOS) und Google Docs (Android) ist das auf jeweils unterschiedlich Art möglich.
Die Frage ergibt sich, ob man schon aus dem Nicht-Vorhandensein bestimmter Menüpunkte in Rotor und TalkBack Menü sicher schließen kann, dass ein Auslesen von Textformaten NICHT unterstützt wird, und ob dies automatisc hein FAIL bedeutet bei nicht editierbarem Text. Ist die Prüfung nur dann sinnvoll, wenn man (wie bei Texteditoren) Formatierung auch ändern kann? Ich verstehe die englische Prüfschrittbeschreibugn der EN so, dass eigentlich immer die Formatierung von Text (auch read-only Text) ausgelesen können werden soll. Ist dies auch in mobilen Platformwen problemlos möglich oder verlangt man her mehr, als ein App-Autor überhaupt umsetzen kann?

**Comment by CarstenKaul** [comment]
Ich würde normalerweise auch den Vorschlag befürworten, Textattribute nur dann als "anwendbar" einzuordnen, wenn entweder der Screenreader ein solches Attribut zum vorlesen anbietet (Rotor) ODER der zu bewertende Text vom Nutzer selbst mit Textattributen versehen werden kann. 
Man müsste dann wiederum unterscheiden, in welcher Anforderung die Zustandsänderung (Textattribut), sowie der aktuelle Status (fett) bemängelt werden soll.
Da ich das Textattribut "durchgestrichen" für sehr wichtig halte, sollte man dies bei der Einschätzung berücksichtigen

**Comment by detlevhfischer** [comment]
Die Unterstützung von `s`  in HTML ist lückenhaft siehe den [Artikel von Webaxe](https://www.webaxe.org/strikethrough-html-accessibility/) dazu) -  ich vermute, nativ wird sie kaum besser sein. Wenn strikethrough Text wichtige Infos bietet (wie bei alter Preis / neuer Preis, kommen Entwickler wohl nicht umhin, über den eine Ergänzung im zugänglichen Namen (z.B. "alter Preis: 400 Euro") wo visuell ~~400,00 €~~ steht) die Semantik nachzubauen. Und wo das nicht geschieht, sehe ich ein 11.5.2.10 FAIL).

**Comment by detlevhfischer** [comment]
@CarstenKaul Können wir das Issue schließen?

**Comment by CarstenKaul** [comment]
Ja, danke
